app/cells/members/row_cell.rb

```diff
@@ -149,6 +144,21 @@ def column_css_class(column)
       end
     end
 
+    def principal_link
+      case Principal
+      when User
+        link_to principal.name, user_path(principal)
+      when Group
+        link_to principal.name, show_group_path(principal)
+      else
+        content_tag :span, principal.name
```

Comment:
We need to revisit this once the placeholder users administration PR (#8944) is in.